Code in guide "Scripting with Quarkus" does not run with jbang

### Describe the bug

The code https://quarkus.io/guides/scripting#solution can't be run with `jbang`. A stracktrace is shown. As soon as the dependency `io.quarkus:quarkus-resteasy-reactive:2.8.0.Final` is replaced by `io.quarkus:quarkus-resteasy:2.8.0.Final` the script runs.

### Expected behavior

Application startup without a stacktrace showing the Quarkus banner and the installed features. Running `curl http://localhost:8080/hello` prints `hello`

### Actual behavior

The following stacktrace is shown

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/eclipse/microprofile/config/ConfigValue
	at io.smallrye.config.SmallRyeConfigBuilder.build(SmallRyeConfigBuilder.java:449)
	at io.quarkus.runtime.generated.Config.<clinit>(Unknown Source)
	at io.quarkus.runner.ApplicationImpl.<clinit>(Unknown Source)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128)
	at java.base/jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:347)
	at java.base/java.lang.Class.newInstance(Class.java:645)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:66)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:41)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:120)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
Caused by: java.lang.ClassNotFoundException: org.eclipse.microprofile.config.ConfigValue
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 14 more
```

### How to Reproduce?

1. install JBang; e.g. `curl -Ls https://sh.jbang.dev | bash -s - app setup`
2. save the code from https://quarkus.io/guides/scripting#solution in a file called `quarkusapp.java`
3. run the code using `jbang run quarkusapp.java`

### Output of `uname -a` or `ver`

Darwin Kernel Version 21.4.0

### Output of `java -version`

ava version "17" 2021-09-14 LTS Java(TM) SE Runtime Environment (build 17+35-LTS-2724) Java HotSpot(TM) 64-Bit Server VM (build 17+35-LTS-2724, mixed mode, sharing)

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.8.0.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

jbang version 0.92.2

### Additional information

_No response_

--- a/docs/src/main/asciidoc/scripting.adoc
+++ b/docs/src/main/asciidoc/scripting.adoc
@@ -28,7 +28,8 @@ Normally we would link to a Git repository to clone but in this case there is no
 [source,java,subs=attributes+]
 ----
 //usr/bin/env jbang "$0" "$@" ; exit $?
-//DEPS io.quarkus:quarkus-resteasy-reactive:{quarkus-version}
+//DEPS io.quarkus.platform:quarkus-bom:{quarkus-version}@pom
+//DEPS io.quarkus:quarkus-resteasy-reactive
 //JAVAC_OPTIONS -parameters
 //JAVA_OPTIONS -Djava.util.logging.manager=org.jboss.logmanager.LogManager
 
@@ -105,20 +106,21 @@ You will find at the top a line looking like this:
 
 This line is what on Linux and macOS allows you to run it as a script. On Windows this line is ignored.
 
-The next line
+The next lines
 
 [source,java]
 ----
 // //DEPS <dependency1> <dependency2>
 ----
 
-Is illustrating how you add dependencies to this script. This is a feature of JBang.
+illustrate how you add dependencies to this script. This is a feature of JBang.
 
-Go ahead and update this line to include the `quarkus-resteasy-reactive` dependency like so:
+Go ahead and update this line to include the `quarkus-bom` and the `quarkus-resteasy-reactive` dependency like so:
 
 [source,java,subs=attributes+]
 ----
-//DEPS io.quarkus:quarkus-resteasy-reactive:{quarkus-version}
+//DEPS io.quarkus.platform:quarkus-bom:{quarkus-version}@pom
+//DEPS io.quarkus:quarkus-resteasy-reactive
 ----
 
 Now, run `jbang quarkusapp.java` and you will see JBang resolving this dependency and building the jar with help from Quarkus' JBang integration.

